internal/buildscripts/packaging/choco/splunk-otel-collector/tools/chocolateyinstall.ps1

```diff
@@ -210,7 +206,7 @@ if (!$env_vars["SPLUNK_ACCESS_TOKEN"]) {
             # The fluentd service is automatically started after msi installation.
             # Wait for it to be running before trying to restart it with our custom config.
             Write-Host "Restarting $fluentd_service_name service..."
-            wait_for_service -name "$fluentd_service_name"
+            start_service -name "$fluentd_service_name"

```

Comment:
the comment above indicates that fluentd service will be started automatically. should the comment be updated or is this start_service safe to run in place of just waiting?